theme-selector.js

```diff
@@ -69,6 +71,72 @@ class ThemeSelector extends HTMLElement {
 
     return template.content
   }
+
+  createColorBlindFilters () {
+    const svgNS = 'http://www.w3.org/2000/svg'
+    const svg = document.createElementNS(svgNS, 'svg')
+    svg.setAttribute('id', 'colorblind-filters')
+    svg.setAttribute('style', 'display: none')
+
+    const defs = document.createElementNS(svgNS, 'defs')
+
+    const filters = [
+      {
+        id: 'deuteranopia',
+        values: '0.29031,0.70969,0.00000,0,0 0.29031,0.70969,0.00000,0,0 -0.02197,0.02197,1.00000,0,0 0,0,0,1,0'
+      },
+      {
+        id: 'deuteranomaly',
+        values: '0.57418,0.42582,0.00000,0,0 0.17418,0.82582,0.00000,0,0 -0.01318,0.01318,1.00000,0,0 0,0,0,1,0'
+      },
+      {
+        id: 'protanopia',
+        values: '0.10889,0.89111,0.00000,0,0 0.10889,0.89111,0.00000,0,0 0.00447,-0.00447,1.00000,0,0 0,0,0,1,0'
+      },
+      {
+        id: 'protanomaly',
+        values: '0.46533,0.53467,0.00000,0,0 0.06533,0.93467,0.00000,0,0 0.00268,-0.00268,1.00000,0,0 0,0,0,1,0'
+      },
+      {
+        id: 'tritanopia',
+        values: '1.00000,0.15236,-0.15236,0,0 0.00000,0.86717,0.13283,0,0 0.00000,0.86717,0.13283,0,0 0,0,0,1,0'
+      },
+      {
+        id: 'tritanomaly',
+        values: '1.00000,0.09142,-0.09142,0,0 0.00000,0.92030,0.07970,0,0 0.00000,0.52030,0.47970,0,0 0,0,0,1,0'
+      },
+      {
+        id: 'achromatopsia',
+        values: '0.299,0.587,0.114,0,0 0.299,0.587,0.114,0,0 0.299,0.587,0.114,0,0 0,0,0,1,0'
+      }
+    ]
+
```

Comment:
I've used Colorblindly for the themes.
Extension: https://chromewebstore.google.com/detail/colorblindly/floniaahmccleoclneebhhmnjgdfijgg
GitHub: https://github.com/oftheheadland/Colorblindly

The media element filters are accurate:
![color-blind_media_filters](https://github.com/hyphacoop/reader.distributed.press/assets/68826419/7a69de0e-24ef-4bd3-9491-78e2a1e90592)

https://github.com/oftheheadland/Colorblindly/blob/master/script/hover.js